doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,372 @@
+
+If the Upper Transport PDU fits, it is sent in a single Lower Transport PDU.
+An Upper Transport PDU that fits into one Lower Transport PDU can also be sent as a single-segment segmented message when acknowledgment
+by the lower transport layer is required.
+
+If the Upper Transport PDU doesn’t fit, the lower transport layer divides the Upper Transport PDU into two or more Lower Transport PDUs.
+If a segmented Upper Transport PDU is tagged with additional metadata, each Lower Transport PDU of the segmented Upper Transport PDU is
+tagged with the same metadata.
+
+Delivery of a segmented message is acknowledged by the lower transport layer of the receiving node.
+Delivery of an unsegmented message is not acknowledged.
+
+The process of segmentation for Upper Transport Access PDUs and Upper Transport Control PDUs is identical, but the segment sizes are different
+for the two PDU types.
+For Upper Transport Access PDU, each segment is 12 octets long, with the exception of the last segment which may be shorter.
+For Upper Transport Control PDU, each segment is 8 octets long, with the exception of the last segment which may be shorter.
+
+The following fields and values are important when handling segments:
+
+SEQ
+   The ``SEQ`` field holds the sequence number.
+
+SegO
+   Each segment of an Upper Transport PDU is identified by the ``SegO`` field value.
+   The ``SegO`` field value of the first segment equals 0, and of the last segment equals the ``SegN`` field value.
+
+SegN
+   The total number of segments is identified by the ``SegN`` field value.
+
+SeqAuth
+   Lower Transport PDUs derived from the same Upper Transport PDU are linked together by the common ``SeqAuth`` value.
+   The ``SeqAuth`` value is composed of the IV Index and the sequence number of the first segment.
+   All Lower Transport PDUs from the same Upper Transport PDU are sent using the same IV Index from the common ``SeqAuth``.
+
+SeqZero
+   The ``SeqZero`` field is included in the segmented message and Segment Acknowledgment message to identify the Upper Transport PDU.
+   The least significant 13 bits of the ``SeqAuth`` value constitute the ``SeqZero`` field value.
+
+Transmission
+============
+
+Never more than one segmented message is transmitted to the same destination.
+The segments are sent in order of ``SegO`` field value, starting with the lowest value.
+Transmission of segments is separated by a segment transmission interval (value indicated by the SAR Segment Interval Step state).
+
+It is not possible to send a segmented message when the ``SEQ`` field value is 8192 greater than the ``SeqAuth`` value.
+If a segmented message has not been acknowledged by the time that the ``SEQ`` field gets this value, the transmission of the Upper
+Transport PDU is canceled and the Lower Transport PDUs are delivered to the network layer.
+
+Segments can be sent to a unicast address or to a group or a virtual address.
+
```

Comment:
Is it possible to merge the "Transmitting to a ...." into one and just explain what it unique for unicast address, as there is a lot of repeating text. It makes it less to read and maintain. Same for retransmition.